### PR TITLE
Added new abbrevation 1. htf - how the fuck

### PR DESCRIPTION
**Pull request by PriyansuMaurya**
2. ps - problem statement, post script
3. td - to do
4. lnkd - linkedin
5. tuh - thank you
6. imao - In my arrogant opinion

and event renamed this s%2Fo.json ==> so.json

Closes #

### What new changes did you make? Tick all applicable boxes
- [x] Added new abbreviation
- [x] Fixed something in the source code
- [ ] Added a new feature
- [ ] Fixed the docs (README.md, CONTRIBTUING.md, etc)

### CODE REVIEW

**Review by Njong392 — CHANGES_REQUESTED** [review]
Thank you for the new changes! Loving everything you added, except the filename change you made. That was added because say a user wanted to add `s/o.json` to the records. It won'y work because `/` is  not accepted in filenames. The filename you changed was the encoded version for `s/o`/... I don't know why it doesn't work... :thinking: weird. But please, revert it for the sake of reverting. Thanks.

**Comment by E-fais** [comment]
LGTM

**Comment by Njong392** [comment]
@PriyansuMaurya how is this going?

**Comment by Njong392** [comment]
No problems. You can get back to it whenever. Was just checking in

**Review by Njong392 — CHANGES_REQUESTED** [review]
I have no problems with the new additions, but abbreve is an 'unofficial' dictionary. Unfortunately, words like these cannot be added. Hope you understand.

File: public/server/db/iv.json [hunk 1]
```diff
@@ -0,0 +1,3 @@
+{ 
+     "definition": "Industrial Visit"
```
Comment (Njong392) [inline public/server/db/iv.json:2]: Just like `gsoc` these does not sound like an 'unofficial' abbreviation

File: public/server/db/pyq.json [hunk 2]
```diff
@@ -0,0 +1,3 @@
+{ 
+     "definition": "Previous Year Question"
```
Comment (Njong392) [inline public/server/db/pyq.json:2]: This could also be kicked out

File: public/server/db/ps.json [hunk 3]
```diff
@@ -0,0 +1,5 @@
+{
+    "definition": "Problem Statement",
```
Comment (Njong392) [inline public/server/db/ps.json:2]: Thinking hard about this one... but I really think it could go too.

**Review by Njong392 — CHANGES_REQUESTED** [review]

File: public/server/db/ps.json (hunk 3 above)
Comment (PriyansuMaurya) [reply public/server/db/ps.json:2]: Alright NP, As you wish 🙃.

**Comment by Njong392** [comment]
Super late review, but I think `gsoc` and `linkedin` can also go.

**Comment by PriyansuMaurya** [comment]
> Super late review, but I think `gsoc` and `linkedin` can also go.

This is hardest pull request I have made. 🥲🥲
I'll do it anyway 😜. It's just 2 files.

**Comment by Njong392** [comment]
I'm so sorry to put you through all the trouble! I appreciate you for sticking around even though I review the pull requests too much haha. Thank you for the new changes.

**Review by Njong392 — APPROVED** [review]
Looks perfect now! 🎊 Can I say 'finally'? 🤣

**Pull request merged**